test/steward_multi_token_deposit.js

```diff
@@ -40,25 +51,43 @@ contract('WildcardSteward owed', (accounts) => {
 
   it('steward: deposit-management. On token buy, check that the remaining deposit is sent back to patron only if it is their only token', async () => {
     await waitTillBeginningOfSecond()
-    // buy 2 tokens, with prices of 1 ether and 2 ether.
+
     await steward.buy(testTokenId1, web3.utils.toWei('1', 'ether'), { from: accounts[2], value: web3.utils.toWei('1', 'ether') });
+    await steward.deposit.call(accounts[2]);
+
     await steward.buy(testTokenId2, web3.utils.toWei('2', 'ether'), { from: accounts[2], value: web3.utils.toWei('1', 'ether') });
+    await steward.deposit.call(accounts[2]);
+
+    const priceOftoken1 = await steward.price.call(testTokenId1)
+    const priceOftoken2 = await steward.price.call(testTokenId2)
 
+    // TIME INCREASES HERE BY 10 MIN
     await time.increase(time.duration.minutes(10));
 
     const patronDepositBeforeSale = await steward.deposit.call(accounts[2]);
     const balancePatronBeforeSale = new BN(await web3.eth.getBalance(accounts[2]));
+
+    //First token then bought. Deposit should remain.
     await steward.buy(testTokenId1, ether('1'), { from: accounts[3], value: ether('2') });
-    const balancePatronAfterFirstSale = new BN(await web3.eth.getBalance(accounts[2]));
+
     const patronDepositAfterFirstSale = await steward.deposit.call(accounts[2]);
+    const balancePatronAfterFirstSale = new BN(await web3.eth.getBalance(accounts[2]));
+
+    //Second token then bought. Deposit should now be added back the users balance
     await steward.buy(testTokenId2, ether('1'), { from: accounts[3], value: ether('3') });
+
     const balancePatronAfterSecondSale = new BN(await web3.eth.getBalance(accounts[2]));
     const patronDepositAfterSecondSale = await steward.deposit.call(accounts[2]);
 
-    // NOTE:: These tests are extremely flakey due to timeing difficulties in ganache... (something to look into)
-    assert(patronDepositBeforeSale.sub(patronDepositAfterFirstSale).lt(new BN("684931506849316")));
+    //Patronage of multi tokens after set amount of time.
```

Comment:
I don't think this comment is necessary given the descriptive name of the variable